### PR TITLE
Add limit to invoice ref field

### PR DESCRIPTION
**Pull request by alexBinns**

### CODE REVIEW

File: legacy/src/api/payment-requests/payment-requests.md [hunk 1]
```diff
@@ -919,6 +919,8 @@ One of either `assetId`, `transactionId` or `authorization` must be provided.
 <a name="refund"></a>
 ### Refund a Payment Request
 
+To refund a Payment Request that has been paid with a Token requires the line item to be provided.
```
Comment (daallen303) [inline legacy/src/api/payment-requests/payment-requests.md:922]: This should be a separate PR

Comment (alexBinns) [reply legacy/src/api/payment-requests/payment-requests.md:922]: Oh yea my bad

**Review by daallen303 — APPROVED** [review]
